runtime/src/precompiles/mod.rs

```diff
@@ -13,10 +13,12 @@ use pallet_evm_precompile_simple::{ECRecover, ECRecoverPublicKey, Identity, Ripe
 // Include custom precompiles
 mod balance_transfer;
 mod ed25519;
+mod metagraph;
```

Comment:
`METAGRAPH_PRECOMPILE_INDEX` also needs to be in `used_addresses` so that `is_precompile` works correctly.

Reply:
Added.